constants/constants.go

```diff
@@ -79,4 +79,10 @@ const (
 	StakerCertFileName = "staker.crt"
 	StakerKeyFileName  = "staker.key"
 	BLSKeyFileName     = "signer.key"
+
+	// errors
+	EmptyValidatorNodeIDError   = "validator node id is not provided"
```

Comment:
should be errors objects, not string, and the constant name start with Err. take a look at https://github.com/ava-labs/avalanchego/blob/57fe51dd59a223422d68acfb48f4bc3892cad782/vms/platformvm/utxo/verifier.go#L25